Marlin/src/module/probe.cpp

```diff
@@ -716,9 +716,9 @@ float Probe::probe_at_point(const float &rx, const float &ry, const ProbePtRaise
       if (stow()) measured_z = NAN;   // Error on stow?
 
     if (verbose_level > 2) {
-      SERIAL_ECHOPAIR_F("Bed X: ", LOGICAL_X_POSITION(rx), 3);
```

Comment:
This will now default to 2. 
Is that a desired behavior change?

Reply:
For X position?? Pfft. No one will care, surely.